docs/src/pages/component-api/Tabs/Tabs.md

```diff
@@ -8,17 +8,17 @@ Props
 
 | Name | Type | Default | Description |
 |:-----|:-----|:--------|:------------|
-| buttonClassName | string | | The CSS class name of the manual scroll buttons. |
+| buttonClassName | string |  | The CSS class name of the scroll button elements. |
 | centered | bool | false | If `true`, the tabs will be centered. This property is intended for large views. |
 | children | node |  | The content of the component. |
 | className | string |  | The CSS class name of the root element. |
 | fullWidth | bool | false | If `true`, the tabs will grow to use all the available space. This property is intended for small views. |
-| index | number |  | The index of the currently selected `Tab`. |
+| index | number |  | The index of the currently selected `BottomNavigation`. |
```

Comment:
Ahh, in looking further, I missed updating the same doc in `Tabs.js`.  Now I see that the docs generator is rebuilding the docs from the components.  Silly me, I created the docs by hand 😧 .

I'll get a PR in for fixing `Tabs.js` and rebuilding the docs.